pkg/runtime/runtime.go

```diff
@@ -3150,14 +3297,46 @@ func ShutdownSignal() chan os.Signal {
 	return stop
 }
 
+func (a *DaprRuntime) createHTTPEndpointsChannels() (map[string]channel.HTTPEndpointAppChannel, error) {
+	// Create dedicated app channels for known app endpoints
+	endpoints := a.compStore.ListHTTPEndpoints()
+
+	if len(endpoints) > 0 {
+		channels := make(map[string]channel.HTTPEndpointAppChannel, len(endpoints))
+
+		for _, e := range endpoints {
+			pipeline, err := a.buildAppHTTPPipeline()
+			if err != nil {
+				return nil, fmt.Errorf("failed to build app HTTP pipeline: %w", err)
```

Comment:
Why is this the only error message that is wrapped (with a non-specific message)?

Reply:
Kept from the original code, agree it shouldn't be wrapped here. Changed